### DISCUSSION TITLE
Why is projects tab not being used

### DISCUSSION THREAD

**Discussion by quick007** [Minehut/Meta#450] [Q&A] [answered]
its not ry being used and it would be cool if we could use it

**Comment by MiginsMC** [comment, marked as answer]
wdym it is?

**Reply by quick007** [reply]
![image](https://user-images.githubusercontent.com/69743171/103389255-9a6ede00-4ac2-11eb-8d34-1e73baccf8dd.png)

**Reply by Sniped** [reply]
What is there to update in it? There hasn't been anything planned since that time and nothing has been moved in it since that time.

**Reply by quick007** [reply]
> What is there to update in it? There hasn't been anything planned since that time and nothing has been moved in it since that time.

sure...
![image](https://user-images.githubusercontent.com/69743171/103390602-36e8ae80-4aca-11eb-816a-db0cba6e3cc8.png)

**Reply by quick007** [reply]
also iirc theres other ones as well

**Reply by Sniped** [reply]
Issues pertaining to discord aren't put on the project board, only ones related to game and web panel.

**Reply by quick007** [reply]
this was fixed: https://github.com/Minehut/Meta/projects/2#card-49109254

**Reply by quick007** [reply]
> Issues pertaining to discord aren't put on the project board, only ones related to game and web panel.

why was this not stated 😖

**Reply by Sniped** [reply]
![](https://i.imgur.com/2GJKuUV.png)
